stan/math/prim/scal/prob/beta_binomial_lcdf.hpp

```diff
@@ -103,10 +103,9 @@ namespace stan {
         const T_partials_return mu = alpha_dbl + n_dbl + 1;
         const T_partials_return nu = beta_dbl + N_dbl - n_dbl - 1;
 
-        const T_partials_return F = F32((T_partials_return)1, mu,
-                                                    -N_dbl + n_dbl + 1,
-                                                    n_dbl + 2, 1 - nu,
-                                                    (T_partials_return)1);
+        T_partials_return F;
+        F = F32((T_partials_return)1, mu, -N_dbl + n_dbl + 1, n_dbl + 2,
+          1 - nu, (T_partials_return)1);
```

Comment:
And whatever you do, don't build up `T_partials_return(1)` more than once!  Just set it and assign it if you really need this thing to be a `T_partials_return` type.  In which case, it seems to indicate that the template types in `F32` are broken.